src/components/formSections/components/TermsListField.js

```diff
@@ -120,6 +120,14 @@ export default class TermsListField extends React.Component {
     );
   }
 
+  validateNoteField = (values, termValue) => {
```

Comment:
So why can't we raise the error to redux-form from in here like the other modules do? Would be nice to do it here rather than `FormTerms` knowing about the structure of the terms array and individual elements.

Reply:
Added in the callback onError prop